docs/support/faq/how-to-use-pouta-for-remote-desktop.md

```diff
@@ -1,44 +1,50 @@
 # How to use Pouta for Remote Desktop
 
-This chapter will go through how to set up a remote desktop with noVNC and
-ssh-tunnel on a non-GPU flavor in c- and ePouta. We are using noVNC because
-it allows us to use our browser to access the desktop. If you would like to have
-instruction how to use a GPU instance for rendering you can find instructions
-here [How to use cPouta GPU for rendering](how-to-use-cpouta-gpu-for-rendering.md)
+This article will go through how to set up a remote desktop with noVNC and
+ssh-tunnel into a non-GPU flavor in c- and ePouta. We are using noVNC because
+it allows us to use our browser to access the desktop, i.e. no 
+local installations needed. If you would like to have
+instructions on how to use a GPU instance for rendering look
+here: [How to use cPouta GPU for rendering](how-to-use-cpouta-gpu-for-rendering.md)
 
 [TOC]
 
 ## Preparations
 We will utilize the following technologies to install the remote desktop:
 
-  - A standard-flavor for example standard.medium 
-  - CentOS-7 images (In Pouta it has already the epel-repository installed)
+  - A standard-flavor, for example, _standard.medium_ 
+  - CentOS-7 image (In Pouta they have already the epel-repository installed)
   - snapd to install the noVNC
-  - noVNC this allows to use our browser instead of a traditional VNC-client to view the desktop
-  - tigervnc-server as our VNC server.
-  -  Xfce as our Desktop environment.
-  -  ssh-tunnel so that the VNC server is not open to the internet. This is very
+  - noVNC, since it allows to use our browser instead of a traditional VNC-client to view the desktop
+  - tigervnc-server as our VNC server
+  - Xfce as our Desktop environment
+  - ssh-tunnel so that the VNC server is not open to the internet. This is very
 important.
 
 ## Create and access your instance for remote desktop
 
-1. Launch an standard-flavor instances with the CentOS-7 image.
+1. Launch a standard-flavor instance with the CentOS-7 image.
 2. Attach a floating IP to the instance.
 3. In the security rules allow ingress ssh (port 22).
-4. We will ssh into the instance with this command to create a ssh-tunnel 
-tunnel. Note that the port 2001 is the one that you will use with the browser later.
-
+4. We will ssh into the instance with this command and create a ssh-tunnel. 
 ```
 ssh -L2001:localhost:6081 cloud-user@YOUR-FLOATING-IP
 ```
-
+This also works at least in the Windows Powershell. If you don't have
+an ssh-agent running, you will need to specify also your ssh-key:
+```
+ssh -i C:\users\localusername\.ssh\yourkey.pem -L2001:localhost:6081 cloud-user@YOUR-FLOATING-IP
```

Comment:
Does it work on any fresh windows installation? If not does something more need to be mentioned?